### PR TITLE
ENH: LCLS, BeamStats, Load Order, Scan PVs

### LINKED ISSUE #152: BeamStats
Every hutch should load the `BeamStats` object. Unclear whether this should be a `happi` device at it has no configuration information.

### LINKED ISSUE #189: Scan PVs enable in config file
## Expected Behavior

Allow user to load scan pvs as enabled in the configuration

## Current Behavior

No such allowance, therefore there is always a load line for "scan pvs (disabled)

### LINKED ISSUE #241: Need to include BeamStats and LCLS classes
## Expected Behavior

Every session should have these key information classes from pcdsdevices

## Possible Solution

Add a stage to load_conf

## Context

https://github.com/pcdshub/Bug-Reports-and-Requests/issues/33

### LINKED ISSUE #242: Load questionnaire objects sooner
## Expected Behavior

qs objects should be loaded prior to beamline

## Current Behavior

qs objects loaded very late

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

A bunch of small additions/fixes all in one PR:
- LCLS object
- BeamStats object
- Fix load order so beamline and experiment files happen as late as possible
- Enable scan PVs for all consumers

## Motivation and Context

- closes #152
- closes #189 
- closes #241
- closes #242

## How Has This Been Tested?

TBH I have not tested this yet, one sec

### CODE REVIEW

File: hutch_python/load_conf.py [hunk 1]
```diff
@@ -3,33 +3,33 @@
 The file's specification can be found on the `yaml_files` page.
 """
 import logging
-import yaml
 from copy import copy
 from pathlib import Path
 from socket import gethostname
 from types import SimpleNamespace
 
-from . import mpl_config  # noqa: F401 # isort: ignore
```
Comment (ZLLentz) [inline hutch_python/load_conf.py:12]: isort is being mean...

Comment (klauer) [reply hutch_python/load_conf.py:12]: Try bumping its version in pre-commit to latest (5.6.4) 🤞

Comment (ZLLentz) [reply hutch_python/load_conf.py:12]: No permutations seem to work. There is a difference between where isort moves this line with or without an ignore and with or without a skip, but none of the permutations keep it where it needs to be. I'm going to look into other options for configuring isort.

**Comment by klauer** [comment]
Not again... what's happening with CI? https://travis-ci.com/github/pcdshub/hutch-python/jobs/403506883

**Comment by ZLLentz** [comment]
This is unrelated to the earlier issues I think

**Comment by klauer** [comment]
I think you're right - looks like pcdsdevices v3.1.0 just hasn't reached conda-forge just yet.

**Comment by ZLLentz** [comment]
I mean, this dependency conflict hieroglyph is suggesting that some dependencies don't support 3.6 any more. And this repo can only do 3.6 due to some of the deps that haven't been cleaned up... But I don't think the dependencies suddenly stop supporting 3.6 even in old versions overnight.

**Comment by ZLLentz** [comment]
Maybe the non-conda-forge builds are just bad? And it messed up the dependency resolution?

**Comment by klauer** [comment]
Re-ran after pcdsdevices new version got released - CI is back in business 🥳

File: hutch_python/load_conf.py (hunk 1 above)
Comment (klauer) [reply hutch_python/load_conf.py:12]: Seems to be an isort bug then, I suppose. I'll make an issue to refactor this down the line so we can still use it.

**Comment by ZLLentz** [comment]
Oh thank god

**Comment by ZLLentz** [comment]
@klauer any further thoughts on the PR? I think it's pretty boring/straightforward

**Review by klauer — APPROVED** [review]
Looks good to go, @ZLLentz!

**Pull request merged**